package.json

```diff
@@ -22,19 +22,19 @@
 	"dependencies": {
```

Comment:
Please only update dependencies if strictly necessary - it tends to break things and be more trouble than it's worth. 

Dependabot alerts are disabled in GH but still seem to be happening. Will look into that.